openvdb/CMakeLists.txt

```diff
@@ -36,12 +36,16 @@ cmake_minimum_required(VERSION 3.3)
 if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
+include(GNUInstallDirs)
 
 ###### OpenVDB Core Options
 
 option(OPENVDB_CORE_SHARED "Build dynamically linked version of the core library." ON)
 option(OPENVDB_CORE_STATIC "Build statically linked version of the core library." ON)
 
+set(OPENVDB_INCLUDE_DIR "${CMAKE_INSTALL_INCLUDEDIR}/openvdb" CACHE STRING
+    "The prefix-relative path for installed openvdb headers."
+)
```

Comment:
Any reason you chose to make this a cached item? Do you require the ability to modify this? If we are to use the unified standard (GNUInstallDirs) then ideally these shouldn't be advertised as modifiable

Reply:
Good question -- I can remove the cache var if you prefer.  The reason I did this was because of the slightly-hard-to-maintain feature where we allow the `openvdb_houdini` component to install into a different prefix.

That alone is enough of a departure from the "natural cmake semantics" that it trickles down into needing to expose a lot more to deal with that duality.  IMO that feature can be dropped because it significantly complicates the build scripts, but I did not want to disrupt existing use cases.  Let me know if it's worth keeping it in light of that use case.

Reply:
> IMO that feature can be dropped because it significantly complicates the build scripts

Are you referring to the inclusion of the CACHE variable here or the installation behaviour of `openvdb_houdini`? Either way, I think making the install path of the core library the ground truth (`set(OPENVDB_INCLUDE_DIR "${CMAKE_INSTALL_INCLUDEDIR}/openvdb"`) and allowing more modifications on the install paths of  `openvdb_houdini` is perhaps the way to help resolve this?